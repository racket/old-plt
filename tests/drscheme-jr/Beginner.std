Beginner level tests of DrScheme Jr
Date: Fri Mar 20 14:49:43 CST 1998
DrScheme Jr is loading. Please wait...
Welcome to DrScheme Jr version 51, Copyright (c) 1995-98 PLT
Language: Beginner
|- |- |- |- |- |- |- |- |- 'define
|- (cons "Beginner" (cons "Intermediate" (cons "Advanced" (cons "R4RS+" null))))
|- |- 5
|- 'lambda
|- (cons "Beginner" (cons "Intermediate" (cons "Advanced" (cons "R4RS+" null))))
|- (lambda (a1) ...)
|- 'app
|- (cons "Beginner" (cons "Intermediate" (cons "Advanced" (cons "R4RS+" null))))
|- |- 5
|- 'cond-1
|- (cons "Beginner" (cons "Intermediate" (cons "Advanced" (cons "R4RS+" null))))
|- 5
|- 'cond-2
|- (cons "Beginner" (cons "Intermediate" (cons "Advanced" (cons "R4RS+" null))))
|- 5
|- 'cond-3
|- (cons "Beginner" (cons "Intermediate" (cons "Advanced" (cons "R4RS+" null))))
|- 5
|- 'cond-4
|- (cons "Beginner" (cons "Intermediate" (cons "Advanced" (cons "R4RS+" null))))
|- 5
|- 'if-1
|- (cons "Beginner" (cons "Intermediate" (cons "Advanced" (cons "R4RS+" null))))
|- 5
|- 'if-2
|- (cons "Beginner" (cons "Intermediate" (cons "Advanced" (cons "R4RS+" null))))
|- 5
|- 'quote-1
|- (cons "Beginner" (cons "Intermediate" (cons "Advanced" (cons "R4RS+" null))))
|- 'a
|- 'quote-2
|- (cons "Beginner" (cons "Intermediate" (cons "Advanced" (cons "R4RS+" null))))
|- 'a
|- 'local-1
|- (cons "Intermediate" (cons "Advanced" (cons "R4RS+" null)))
|- stdin[1385-1386]:1.8-1.9
   Empty combination is a syntax error
|- 'local-2
|- (cons "Intermediate" (cons "Advanced" (cons "R4RS+" null)))
|- stdin[1528-1539]:1.9-1.20
   Invalid definition: must be at the top level
|- 'local-3
|- (cons "Intermediate" (cons "Advanced" (cons "R4RS+" null)))
|- stdin[1682-1739]:2.3-5.25
   Invalid definition: must be at the top level
|- 'local-4
|- (cons "Intermediate" (cons "Advanced" (cons "R4RS+" null)))
|- stdin[1834-1910]:2.3-6.28
   Invalid definition: must be at the top level
|- 'let-1
|- (cons "Intermediate" (cons "Advanced" (cons "R4RS+" null)))
|- stdin[1979-1985]:1.6-1.12
   First term after parenthesis is illegal in an application
|- 'let-2
|- null
|- stdin[2087-2106]:1.6-1.25
   First term after parenthesis is illegal in an application
|- 'let-3
|- (cons "Intermediate" (cons "Advanced" (cons "R4RS+" null)))
|- stdin[2251-2271]:1.7-2.13
   First term after parenthesis is illegal in an application
|- 'let-4
|- null
|- stdin[2359-2378]:1.7-1.26
   First term after parenthesis is illegal in an application
|- 'letrec-1
|- (cons "Intermediate" (cons "Advanced" (cons "R4RS+" null)))
|- stdin[2527-2546]:1.9-1.28
   First term after parenthesis is illegal in an application
|- 'letrec-2
|- (cons "Intermediate" (cons "Advanced" (cons "R4RS+" null)))
|- stdin[2693-2829]:1.9-10.36
   First term after parenthesis is illegal in an application
|- 'letrec-3
|- (cons "Beginner" (cons "Intermediate" (cons "Advanced" (cons "R4RS+" null))))
|- stdin[2878-2892]:1.9-2.15
   First term after parenthesis is illegal in an application
|- 'case-sense
|- (cons "Beginner" (cons "Intermediate" (cons "Advanced" (cons "R4RS+" null))))
|- #t
|- 'set!
|- (cons "Advanced" (cons "R4RS+" null))
|- stdin[3117-3123]:1.6-1.12
   First term after parenthesis is illegal in an application
|- 'begin
|- (cons "Advanced" (cons "R4RS+" null))
|- stdin[3254-3258]:1.2-1.6
   reference to undefined identifier: begin
|- 'begin0
|- (cons "Advanced" (cons "R4RS+" null))
|- stdin[3378-3383]:1.2-1.7
   reference to undefined identifier: begin0
|- 'recur-1
|- (cons "Advanced" (cons "R4RS+" null))
|- stdin[3510-3524]:1.8-2.14
   First term after parenthesis is illegal in an application
|- 'recur-2
|- (cons "Advanced" (cons "R4RS+" null))
|- stdin[3663-3675]:2.8-2.20
   First term after parenthesis is illegal in an application
|- 'recur-3
|- (cons "Advanced" (cons "R4RS+" null))
|- stdin[3794-3821]:1.6-1.33
   First term after parenthesis is illegal in an application
|- 'do-1
|- (cons "Advanced" (cons "R4RS+" null))
|- stdin[3938-3949]:1.10-1.21
   '(1 3 5 7 9) is not a symbol
|- 'do-2
|- (cons "Advanced" (cons "R4RS+" null))
|- stdin[4045-4086]:1.5-2.19
   First term after parenthesis is illegal in an application
|- 'call-cc-1
|- (cons "Advanced" (cons "R4RS+" null))
|- stdin[4195-4260]:1.10-3.51
   Malformed lambda
|- 'call-cc-2
|- (cons "Advanced" (cons "R4RS+" null))
|- stdin[4296-4325]:1.6-1.35
   First term after parenthesis is illegal in an application
|- 'let-cc-1
|- (cons "Advanced" (cons "R4RS+" null))
|- stdin[4466-4470]:3.22-3.26
   First term after parenthesis is illegal in an application
|- 'let-cc-2
|- (cons "Advanced" (cons "R4RS+" null))
|- stdin[4585-4602]:1.6-1.23
   First term after parenthesis is illegal in an application
|- 'when-1
|- (cons "Advanced" (cons "R4RS+" null))
|- stdin[4723-4726]:1.2-1.5
   reference to undefined identifier: when
|- 'when-2
|- (cons "Advanced" (cons "R4RS+" null))
|- stdin[4850-4856]:1.6-1.12
   First term after parenthesis is illegal in an application
|- 'unless-1
|- (cons "Advanced" (cons "R4RS+" null))
|- stdin[4990-4995]:1.2-1.7
   reference to undefined identifier: unless
|- 'unless-2
|- (cons "Advanced" (cons "R4RS+" null))
|- stdin[5121-5127]:1.6-1.12
   First term after parenthesis is illegal in an application
|- 'one-armed-if-1
|- (cons "Advanced" (cons "R4RS+" null))
|- stdin[5266-5274]:1.1-1.9
   If must have an else clause
|- 'one-armed-if-2
|- (cons "Advanced" (cons "R4RS+" null))
|- stdin[5390-5396]:1.6-1.12
   First term after parenthesis is illegal in an application
|- 'eq?-1
|- (cons "Advanced" (cons "R4RS+" null))
|- stdin[5526-5534]:1.1-1.9
   eq?: expected symbols as arguments, received 5, and 5
|- 'quote-num-1
|- (cons "Advanced" (cons "R4RS+" null))
|- stdin[5668-5669]:1.1-1.2
   '5 is not a symbol
|- 'quote-num-2
|- (cons "Advanced" (cons "R4RS+" null))
|- stdin[5775-5783]:1.1-1.9
   '5 is not a symbol
|- 'if-3
|- (cons "Advanced" (cons "R4RS+" null))
|- stdin[5879-5879]:1.5-1.5
   value 5 is neither #t nor #f
|- 'cond-1
|- (cons "Advanced" (cons "R4RS+" null))
|- stdin[5994-5994]:1.8-1.8
   value 5 is neither #t nor #f
|- 'thunk
|- (cons "Advanced" (cons "R4RS+" null))
|- stdin[6115-6116]:1.18-1.19
   All procedures must take at least one argument
|- 'anonymous-fun-app
|- (cons "Advanced" (cons "R4RS+" null))
|- stdin[6241-6259]:1.1-1.19
   First term after parenthesis is illegal in an application
|- 'keyword-interface
|- null
|- stdin[6361-6371]:1.1-1.11
   Invalid use of keyword #%interface
|- 'keyword-unit
|- null
|- stdin[6453-6458]:1.1-1.6
   Invalid use of keyword #%unit
|- 'keyword-struct
|- null
|- stdin[6541-6548]:1.1-1.8
   Invalid use of keyword #%struct
|- 'keyword-unquote
|- null
|- stdin[6632-6640]:1.1-1.9
   Invalid use of keyword #%unquote
|- 'keyword-let-macro
|- null
|- stdin[6726-6736]:1.1-1.11
   Invalid use of keyword #%let-macro
|- 'keyword-set!
|- null
|- stdin[6818-6823]:1.1-1.6
   Invalid use of keyword #%set!
|- 'keyword-if
|- null
|- stdin[6902-6905]:1.1-1.4
   Invalid use of keyword #%if
|- 'keyword-quote
|- null
|- stdin[6987-6993]:1.1-1.7
   Invalid use of keyword #%quote
|- 'keyword-begin0
|- null
|- stdin[7076-7083]:1.1-1.8
   Invalid use of keyword #%begin0
|- 'keyword-begin
|- null
|- stdin[7165-7171]:1.1-1.7
   Invalid use of keyword #%begin
|- 'keyword-let
|- null
|- stdin[7251-7255]:1.1-1.5
   Invalid use of keyword #%let
|- 'keyword-let*
|- null
|- stdin[7336-7341]:1.1-1.6
   Invalid use of keyword #%let*
|- 'keyword-letrec*
|- null
|- stdin[7425-7433]:1.1-1.9
   Invalid use of keyword #%letrec*
|- 'keyword-letrec
|- null
|- stdin[7516-7523]:1.1-1.8
   Invalid use of keyword #%letrec
|- 'keyword-lambda
|- null
|- stdin[7606-7613]:1.1-1.8
   Invalid use of keyword #%lambda
|- 'keyword-let-expansion-time
|- null
|- stdin[7708-7727]:1.1-1.20
   Invalid use of keyword #%let-expansion-time
|- 'keyword-define-expansion-time
|- null
|- stdin[7826-7848]:1.1-1.23
   Invalid use of keyword #%define-expansion-time
|- 'keyword-invoke-open-unit
|- null
|- stdin[7942-7959]:1.1-1.18
   Invalid use of keyword #%invoke-open-unit
|- 'keyword-invoke-unit
|- null
|- stdin[8048-8060]:1.1-1.13
   Invalid use of keyword #%invoke-unit
|- 'keyword-compound-unit
|- null
|- stdin[8151-8165]:1.1-1.15
   Invalid use of keyword #%compound-unit
|- 'keyword-unquote-splicing
|- null
|- stdin[8259-8276]:1.1-1.18
   Invalid use of keyword #%unquote-splicing
|- 'keyword-let-id-macro
|- null
|- stdin[8366-8379]:1.1-1.14
   Invalid use of keyword #%let-id-macro
|- 'keyword-define-id-macro
|- null
|- stdin[8472-8488]:1.1-1.17
   Invalid use of keyword #%define-id-macro
|- 'keyword-define-macro
|- null
|- stdin[8578-8591]:1.1-1.14
   Invalid use of keyword #%define-macro
|- 'keyword-case-lambda
|- null
|- stdin[8680-8692]:1.1-1.13
   Invalid use of keyword #%case-lambda
|- 'keyword-let-values
|- null
|- stdin[8780-8791]:1.1-1.12
   Invalid use of keyword #%let-values
|- 'keyword-let*-values
|- null
|- stdin[8880-8892]:1.1-1.13
   Invalid use of keyword #%let*-values
|- 'keyword-letrec*-values
|- null
|- stdin[8984-8999]:1.1-1.16
   Invalid use of keyword #%letrec*-values
|- 'keyword-letrec-values
|- null
|- stdin[9090-9104]:1.1-1.15
   Invalid use of keyword #%letrec-values
|- 'keyword-define-values
|- null
|- stdin[9195-9209]:1.1-1.15
   Invalid use of keyword #%define-values
|- 'keyword-cond
|- null
|- stdin[9291-9296]:1.1-1.6
   Invalid use of keyword #%cond
|- 'keyword-class*/names
|- null
|- stdin[9385-9398]:1.1-1.14
   Invalid use of keyword #%class*/names
|- 'struct-1
|- (cons "Intermediate" (cons "Advanced" (cons "R4RS+" null)))
|- stdin[9532-9544]:1.2-1.14
   reference to undefined identifier: define-struct
|- stdin[9618-9623]:1.7-1.12
   reference to undefined identifier: make-a
|- 'struct-2
|- (cons "Intermediate" (cons "Advanced" (cons "R4RS+" null)))
|- stdin[9801-9802]:1.40-1.41
   Empty combination is a syntax error
|- stdin[9837-9864]:1.7-1.34
   reference to undefined identifier: make-empty-testing-structure
|- 'struct-3
|- (cons "Intermediate" (cons "Advanced" (cons "R4RS+" null)))
|- stdin[10004-10016]:1.2-1.14
   reference to undefined identifier: define-struct
|- stdin[10091-10110]:1.6-1.25
   First term after parenthesis is illegal in an application
|- 'struct-4
|- (cons "Intermediate" (cons "Advanced" (cons "R4RS+" null)))
|- stdin[10255-10267]:1.2-1.14
   reference to undefined identifier: define-struct
|- stdin[10342-10361]:1.6-1.25
   First term after parenthesis is illegal in an application
|- 'struct-5
|- (cons "Intermediate" (cons "Advanced" (cons "R4RS+" null)))
|- stdin[10506-10518]:1.2-1.14
   reference to undefined identifier: define-struct
|- stdin[10593-10610]:1.6-1.23
   First term after parenthesis is illegal in an application
|- 'delay-1
|- (cons "Intermediate" (cons "Advanced" (cons "R4RS+" null)))
|- stdin[10756-10760]:1.2-1.6
   reference to undefined identifier: delay
|- 'delay-2
|- (cons "Intermediate" (cons "Advanced" (cons "R4RS+" null)))
|- stdin[10932-10936]:1.29-1.33
   First term after parenthesis is illegal in an application
|- 'delay-3
|- (cons "Intermediate" (cons "Advanced" (cons "R4RS+" null)))
|- stdin[11078-11105]:1.7-2.21
   First term after parenthesis is illegal in an application
|- 'quote-list-1
|- (cons "Intermediate" (cons "Advanced" (cons "R4RS+" null)))
|- stdin[11244-11258]:1.1-1.15
   '(a a a) is not a symbol
|- 'quote-list-2
|- (cons "Intermediate" (cons "Advanced" (cons "R4RS+" null)))
|- stdin[11383-11390]:1.1-1.8
   '(a a a) is not a symbol
|- 'empty-list-1
|- (cons "Intermediate" (cons "Advanced" (cons "R4RS+" null)))
|- stdin[11521-11523]:1.1-1.3
   '() is not a symbol
|- 'empty-list-2
|- (cons "Intermediate" (cons "Advanced" (cons "R4RS+" null)))
|- stdin[11654-11663]:1.1-1.10
   '() is not a symbol
|- 'back-quote-1
|- (cons "Intermediate" (cons "Advanced" (cons "R4RS+" null)))
|- stdin[11788-11795]:1.1-1.8
   '(a a a) is not a symbol
|- 'back-quote-2
|- (cons "Intermediate" (cons "Advanced" (cons "R4RS+" null)))
|- stdin[11936-11941]:1.11-1.16
   '(a a) is not a symbol
|- 'back-quote-3
|- (cons "Intermediate" (cons "Advanced" (cons "R4RS+" null)))
|- (cons 'a (cons 'a (cons 'a null)))
|- 'quasi-quote-1
|- (cons "Intermediate" (cons "Advanced" (cons "R4RS+" null)))
|- 'a
|- 'quasi-quote-2
|- (cons "Intermediate" (cons "Advanced" (cons "R4RS+" null)))
|- stdin[12270-12289]:1.1-1.20
   '(a a a) is not a symbol
|- 'quasi-quote-3
|- (cons "Intermediate" (cons "Advanced" (cons "R4RS+" null)))
|- stdin[12439-12451]:1.30-1.42
   '(a a) is not a symbol
|- 'quasi-quote-4
|- (cons "Intermediate" (cons "Advanced" (cons "R4RS+" null)))
|- stdin[12570-12582]:1.22-1.34
   '(a a) is not a symbol
|- 'set!-id
|- "R4RS+"
|- stdin[12631-12634]:1.2-1.5
   reference to undefined identifier: set!
|- stdin[12703-12727]:1.1-1.25
   reference to undefined identifier: some-undefined-identifier
|- 'non-list
|- "R4RS+"
|- stdin[12822-12831]:1.1-1.10
   cons: second argument must be of type <list>, given 3 and 4
|- 'fallthrough-cond-1
|- "R4RS+"
|- stdin[12957-12970]:1.8-1.21
   no matching cond clause
|- 'fallthrough-cond-2
|- "R4RS+"
|- stdin[13058-13067]:1.17-1.26
   First term after parenthesis is illegal in an application
|- 