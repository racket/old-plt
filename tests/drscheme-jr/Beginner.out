Beginner level tests of DrScheme Jr
Date: Fri May 15 10:43:27 CDT 1998
DrScheme Jr is loading. Please wait...
Welcome to DrScheme Jr version 52/3, Copyright (c) 1995-98 PLT
Language: Beginner
> > > > > > > > > 'define
> (cons "Beginner" (cons "Intermediate" (cons "Advanced" (cons "R4RS+" null))))
> > 5
> 'lambda
> (cons "Beginner" (cons "Intermediate" (cons "Advanced" (cons "R4RS+" null))))
> (lambda (a1) ...)
> 'app
> (cons "Beginner" (cons "Intermediate" (cons "Advanced" (cons "R4RS+" null))))
> > 5
> 'cond-1
> (cons "Beginner" (cons "Intermediate" (cons "Advanced" (cons "R4RS+" null))))
> 5
> 'cond-2
> (cons "Beginner" (cons "Intermediate" (cons "Advanced" (cons "R4RS+" null))))
> 5
> 'cond-3
> (cons "Beginner" (cons "Intermediate" (cons "Advanced" (cons "R4RS+" null))))
> 5
> 'cond-4
> (cons "Beginner" (cons "Intermediate" (cons "Advanced" (cons "R4RS+" null))))
> 5
> 'if-1
> (cons "Beginner" (cons "Intermediate" (cons "Advanced" (cons "R4RS+" null))))
> 5
> 'if-2
> (cons "Beginner" (cons "Intermediate" (cons "Advanced" (cons "R4RS+" null))))
> 5
> 'quote-1
> (cons "Beginner" (cons "Intermediate" (cons "Advanced" (cons "R4RS+" null))))
> 'a
> 'quote-2
> (cons "Beginner" (cons "Intermediate" (cons "Advanced" (cons "R4RS+" null))))
> 'a
> 'local-1
> (cons "Intermediate" (cons "Advanced" (cons "R4RS+" null)))
> stdin: 70.8-70.9: Empty combination is a syntax error
> 'local-2
> (cons "Intermediate" (cons "Advanced" (cons "R4RS+" null)))
> stdin: 4.9-4.20: Invalid definition: must be at the top level
> 'local-3
> (cons "Intermediate" (cons "Advanced" (cons "R4RS+" null)))
> stdin: 5.3-8.25: Invalid definition: must be at the top level
> 'local-4
> (cons "Intermediate" (cons "Advanced" (cons "R4RS+" null)))
> stdin: 6.3-10.28: Invalid definition: must be at the top level
> 'let-1
> (cons "Intermediate" (cons "Advanced" (cons "R4RS+" null)))
> stdin: 4.6-4.12: First term after parenthesis is illegal in an application
> 'let-2
> null
> stdin: 4.6-4.25: First term after parenthesis is illegal in an application
> 'let-3
> (cons "Intermediate" (cons "Advanced" (cons "R4RS+" null)))
> stdin: 4.7-5.13: First term after parenthesis is illegal in an application
> 'let-4
> null
> stdin: 4.7-4.26: First term after parenthesis is illegal in an application
> 'letrec-1
> (cons "Intermediate" (cons "Advanced" (cons "R4RS+" null)))
> stdin: 4.9-4.28: First term after parenthesis is illegal in an application
> 'letrec-2
> (cons "Intermediate" (cons "Advanced" (cons "R4RS+" null)))
> stdin: 4.9-13.36: First term after parenthesis is illegal in an application
> 'letrec-3
> (cons "Intermediate" (cons "Advanced" (cons "R4RS+" null)))
> stdin: 4.9-5.15: First term after parenthesis is illegal in an application
> 'case-sense
> (cons "Beginner" (cons "Intermediate" (cons "Advanced" (cons "R4RS+" null))))
> #t
> 'set!
> (cons "Advanced" (cons "R4RS+" null))
> stdin: 9.6-9.12: First term after parenthesis is illegal in an application
> 'begin
> (cons "Advanced" (cons "R4RS+" null))
> stdin: 4.2-4.6: reference to undefined identifier: begin
> 'begin0
> (cons "Advanced" (cons "R4RS+" null))
> stdin: 4.2-4.7: reference to undefined identifier: begin0
> 'recur-1
> (cons "Advanced" (cons "R4RS+" null))
> stdin: 4.8-5.14: First term after parenthesis is illegal in an application
> 'recur-2
> (cons "Advanced" (cons "R4RS+" null))
> stdin: 5.8-5.20: First term after parenthesis is illegal in an application
> 'recur-3
> (cons "Advanced" (cons "R4RS+" null))
> stdin: 4.6-4.33: First term after parenthesis is illegal in an application
> 'do-1
> (cons "Advanced" (cons "R4RS+" null))
> stdin: 4.10-4.21: '`(1 3 5 7 9) is not a symbol
> 'do-2
> (cons "Advanced" (cons "R4RS+" null))
> stdin: 4.5-5.19: First term after parenthesis is illegal in an application
> 'call-cc-1
> (cons "Advanced" (cons "R4RS+" null))
> stdin: 4.10-6.51: Malformed lambda
> 'call-cc-2
> (cons "Advanced" (cons "R4RS+" null))
> stdin: 4.6-4.35: First term after parenthesis is illegal in an application
> 'let-cc-1
> (cons "Advanced" (cons "R4RS+" null))
> stdin: 6.22-6.26: First term after parenthesis is illegal in an application
> 'let-cc-2
> (cons "Advanced" (cons "R4RS+" null))
> stdin: 4.6-4.23: First term after parenthesis is illegal in an application
> 'when-1
> (cons "Advanced" (cons "R4RS+" null))
> stdin: 4.2-4.5: reference to undefined identifier: when
> 'when-2
> (cons "Advanced" (cons "R4RS+" null))
> stdin: 4.6-4.12: First term after parenthesis is illegal in an application
> 'unless-1
> (cons "Advanced" (cons "R4RS+" null))
> stdin: 4.2-4.7: reference to undefined identifier: unless
> 'unless-2
> (cons "Advanced" (cons "R4RS+" null))
> stdin: 4.6-4.12: First term after parenthesis is illegal in an application
> 'one-armed-if-1
> (cons "Advanced" (cons "R4RS+" null))
> stdin: 4.1-4.9: If must have an else clause
> 'one-armed-if-2
> (cons "Advanced" (cons "R4RS+" null))
> stdin: 4.6-4.12: First term after parenthesis is illegal in an application
> 'eq?-1
> (cons "Advanced" (cons "R4RS+" null))
> stdin: 4.1-4.9: eq?: expected symbols as arguments, received 5, and 5
> 'quote-num-1
> (cons "Advanced" (cons "R4RS+" null))
> stdin: 4.1-4.2: '5 is not a symbol
> 'quote-num-2
> (cons "Advanced" (cons "R4RS+" null))
> stdin: 4.1-4.9: '5 is not a symbol
> 'if-3
> (cons "Advanced" (cons "R4RS+" null))
> stdin: 4.5-4.5: if: value 5 is neither #t nor #f
> 'cond-1
> (cons "Advanced" (cons "R4RS+" null))
> stdin: 4.8-4.8: if: value 5 is neither #t nor #f
> 'thunk
> (cons "Advanced" (cons "R4RS+" null))
> stdin: 4.18-4.19: All procedures must take at least one argument
> 'anonymous-fun-app
> (cons "Advanced" (cons "R4RS+" null))
> stdin: 4.1-4.19: First term after parenthesis is illegal in an application
> 'keyword-interface
> null
> stdin: 6.1-6.11: unknown-phase: Invalid use of keyword #%interface
> 'keyword-unit
> null
> stdin: 4.1-4.6: unknown-phase: Invalid use of keyword #%unit
> 'keyword-struct
> null
> stdin: 4.1-4.8: unknown-phase: Invalid use of keyword #%struct
> 'keyword-unquote
> null
> stdin: 4.1-4.9: Invalid use of keyword #%unquote
> 'keyword-let-macro
> null
> stdin: 4.1-4.11: Invalid use of keyword #%let-macro
> 'keyword-set!
> null
> stdin: 4.1-4.6: unknown-phase: Invalid use of keyword #%set!
> 'keyword-if
> null
> stdin: 4.1-4.4: Invalid use of keyword #%if
> 'keyword-quote
> null
> stdin: 4.1-4.7: Invalid use of keyword #%quote
> 'keyword-begin0
> null
> stdin: 4.1-4.8: unknown-phase: Invalid use of keyword #%begin0
> 'keyword-begin
> null
> stdin: 4.1-4.7: unknown-phase: Invalid use of keyword #%begin
> 'keyword-let
> null
> stdin: 4.1-4.5: unknown-phase: Invalid use of keyword #%let
> 'keyword-let*
> null
> stdin: 4.1-4.6: unknown-phase: Invalid use of keyword #%let*
> 'keyword-letrec*
> null
> stdin: 4.1-4.9: unknown-phase: Invalid use of keyword #%letrec*
> 'keyword-letrec
> null
> stdin: 4.1-4.8: unknown-phase: Invalid use of keyword #%letrec
> 'keyword-lambda
> null
> stdin: 4.1-4.8: Invalid use of keyword #%lambda
> 'keyword-let-expansion-time
> null
> stdin: 4.1-4.20: unknown-phase: Invalid use of keyword #%let-expansion-time
> 'keyword-define-expansion-time
> null
> stdin: 4.1-4.23: unknown-phase: Invalid use of keyword #%define-expansion-time
> 'keyword-invoke-open-unit
> null
> stdin: 4.1-4.18: unknown-phase: Invalid use of keyword #%invoke-open-unit
> 'keyword-invoke-unit
> null
> stdin: 4.1-4.13: unknown-phase: Invalid use of keyword #%invoke-unit
> 'keyword-compound-unit
> null
> stdin: 4.1-4.15: unknown-phase: Invalid use of keyword #%compound-unit
> 'keyword-unquote-splicing
> null
> stdin: 4.1-4.18: Invalid use of keyword #%unquote-splicing
> 'keyword-let-id-macro
> null
> stdin: 4.1-4.14: unknown-phase: Invalid use of keyword #%let-id-macro
> 'keyword-define-id-macro
> null
> stdin: 4.1-4.17: unknown-phase: Invalid use of keyword #%define-id-macro
> 'keyword-define-macro
> null
> stdin: 4.1-4.14: Invalid use of keyword #%define-macro
> 'keyword-case-lambda
> null
> stdin: 4.1-4.13: Invalid use of keyword #%case-lambda
> 'keyword-let-values
> null
> stdin: 4.1-4.12: unknown-phase: Invalid use of keyword #%let-values
> 'keyword-let*-values
> null
> stdin: 4.1-4.13: unknown-phase: Invalid use of keyword #%let*-values
> 'keyword-letrec*-values
> null
> stdin: 4.1-4.16: unknown-phase: Invalid use of keyword #%letrec*-values
> 'keyword-letrec-values
> null
> stdin: 4.1-4.15: unknown-phase: Invalid use of keyword #%letrec-values
> 'keyword-define-values
> null
> stdin: 4.1-4.15: Invalid use of keyword #%define-values
> 'keyword-cond
> null
> stdin: 4.1-4.6: Invalid use of keyword #%cond
> 'keyword-class*/names
> null
> stdin: 4.1-4.14: unknown-phase: Invalid use of keyword #%class*/names
> 'struct-1
> (cons "Intermediate" (cons "Advanced" (cons "R4RS+" null)))
> stdin: 6.2-6.14: reference to undefined identifier: define-struct
> stdin: 1.7-1.12: reference to undefined identifier: make-a
> 'struct-2
> (cons "Intermediate" (cons "Advanced" (cons "R4RS+" null)))
> stdin: 4.40-4.41: Empty combination is a syntax error
> stdin: 1.7-1.34: reference to undefined identifier: make-empty-testing-structure
> 'struct-3
> (cons "Intermediate" (cons "Advanced" (cons "R4RS+" null)))
> stdin: 4.2-4.14: reference to undefined identifier: define-struct
> stdin: 1.6-1.25: First term after parenthesis is illegal in an application
> 'struct-4
> (cons "Intermediate" (cons "Advanced" (cons "R4RS+" null)))
> stdin: 4.2-4.14: reference to undefined identifier: define-struct
> stdin: 1.6-1.25: First term after parenthesis is illegal in an application
> 'struct-5
> (cons "Intermediate" (cons "Advanced" (cons "R4RS+" null)))
> stdin: 4.2-4.14: reference to undefined identifier: define-struct
> stdin: 1.6-1.23: First term after parenthesis is illegal in an application
> 'delay-1
> (cons "Advanced" (cons "R4RS+" null))
> stdin: 4.2-4.6: reference to undefined identifier: delay
> 'delay-2
> (cons "Advanced" (cons "R4RS+" null))
> stdin: 4.29-4.33: First term after parenthesis is illegal in an application
> 'delay-3
> (cons "Advanced" (cons "R4RS+" null))
> stdin: 4.7-5.21: First term after parenthesis is illegal in an application
> 'quote-list-1
> (cons "Advanced" (cons "R4RS+" null))
> stdin: 4.1-4.15: '`(a a a) is not a symbol
> 'quote-list-2
> (cons "Advanced" (cons "R4RS+" null))
> stdin: 4.1-4.8: '`(a a a) is not a symbol
> 'empty-list-1
> (cons "Advanced" (cons "R4RS+" null))
> stdin: 4.1-4.3: '`() is not a symbol
> 'empty-list-2
> (cons "Advanced" (cons "R4RS+" null))
> stdin: 4.1-4.10: '`() is not a symbol
> 'back-quote-1
> (cons "Advanced" (cons "R4RS+" null))
> stdin: 4.1-4.8: '`(a a a) is not a symbol
> 'back-quote-2
> (cons "Advanced" (cons "R4RS+" null))
> stdin: 4.11-4.16: '`(a a) is not a symbol
> 'back-quote-3
> (cons "Beginner" (cons "Intermediate" (cons "Advanced" (cons "R4RS+" null))))
> (cons 'a (cons 'a (cons 'a null)))
> 'quasi-quote-1
> (cons "Beginner" (cons "Intermediate" (cons "Advanced" (cons "R4RS+" null))))
> 'a
> 'quasi-quote-2
> (cons "Advanced" (cons "R4RS+" null))
> stdin: 12.1-12.20: '`(a a a) is not a symbol
> 'quasi-quote-3
> (cons "Advanced" (cons "R4RS+" null))
> stdin: 4.30-4.42: '`(a a) is not a symbol
> 'quasi-quote-4
> (cons "Advanced" (cons "R4RS+" null))
> stdin: 4.22-4.34: '`(a a) is not a symbol
> 'set!-id
> null
> stdin: 4.2-4.5: reference to undefined identifier: set!
> stdin: 1.1-1.25: reference to undefined identifier: some-undefined-identifier
> 'non-list
> "R4RS+"
> stdin: 4.1-4.10: cons: second argument must be of type <list>, given 3 and 4
> 'fallthrough-cond-1
> null
> stdin: 4.8-4.21: no matching cond clause
> 'fallthrough-cond-2
> null
> stdin: 4.17-4.26: First term after parenthesis is illegal in an application
> 